doc/whats_new/v0.24.rst

```diff
@@ -113,6 +113,10 @@ Changelog
   argument `rotation`, which can take the value `None`, `'varimax'` or `'quartimax'.`
   :pr:`11064` by :user:`Jona Sassenhagen <jona-sassenhagen>`.
 
+- |Fix| :func:`decomposition.IncrementalPCA.partial_fit` handles overflow error
+  in mean correction.`
```

Comment:
maybe "prevent overflow in :func:... for large `batch_size` and `n_samples` values?

Reply:
sounds better. I'll make the change.